src/v/coproc/types.h

```diff
@@ -41,7 +41,7 @@ enum class enable_response_code : int8_t {
 enum class disable_response_code : int8_t {
     success = 0,
     internal_error,
-    script_id_does_not_exist,
+    script_id_does_not_exist = 7
```

Comment:
this sounds hard to maintain. how many types of errors are there maybe the "code bloat" is worth it? are any of these values transmitted over the network?

also, did you mean to write something different than `coproc::enum` in the commit message?